DxDispatch/src/dxdispatch/HlslDispatchable.cpp

```diff
@@ -438,7 +442,8 @@ void HlslDispatchable::Bind(const Bindings& bindings, uint32_t iteration)
         }
         else if (bindPoint.descriptorType == D3D12_DESCRIPTOR_RANGE_TYPE_CBV)
         {
-            if (sourceBufferDesc.sizeInBytes % D3D12_CONSTANT_BUFFER_DATA_PLACEMENT_ALIGNMENT != 0)
+            if ((sourceBufferDesc.sizeInBytes % D3D12_CONSTANT_BUFFER_DATA_PLACEMENT_ALIGNMENT != 0) ||
+                (sourceBufferDesc.sizeInBytes > std::numeric_limits<uint32_t>::max()))
```

Comment:
Confusing if the error says the buffer size isn't aligned by instead it's just too big.